fabric-gametest-api-v1/build.gradle

```diff
@@ -12,7 +12,6 @@ loom {
 			server()
 			name "Game Test"
 			vmArg "-Dfabric-api.gametest"
-			vmArg "-Dfabric-api.gametest.report-file=${project.buildDir}/junit.xml"
```

Comment:
Hehe oops let me reverse this. Forgot I removed this bit locally due to issue testing on going into multplayer due to weird bugs from a version of the library that I adpated into this codebase!